desktop/core/src/desktop/api2.py

```diff
@@ -1478,3 +1478,14 @@ def _setup_search_examples(request):
 
   if data == 'log_analytics_demo':
     search_setup.Command().handle()
+
+
+@api_error_handler
+def check_config(request):
+  """Returns the configuration directory and the list of validation errors."""
+  response = {
+    'hue_config_dir': os.path.realpath(os.getenv("HUE_CONF_DIR", get_desktop_root("conf"))),
+    'config_error_list': _get_config_errors(request, cache=False),
```

Comment:
Nit: `config_error_list` -> `config_errors`

Reply:
Raised a separate PR for this as this one got auto-merged: https://github.com/cloudera/hue/pull/4019